### PR TITLE
Add initial Quick Launches overview

### PR DESCRIPTION
**Pull request by psarando**

### CODE REVIEW

File: api/endpoints/quick-launches.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+endpoint when launching a job.
+Note that not all required analysis parameters of the `AnalysisSubmissionConfig`
+need to be specified in the Quick Launch.
+
+Once a Quick Launch is defined, the
+[GET /terrain/quicklaunches](https://de.cyverse.org/terrain/docs/index.html#!/analyses-quicklaunches/get_terrain_quicklaunches)
+endpoint can be used to list Quick Launches that are available to the user.
+The description currently says that it lists Quick Launches that were created by the user.
+This isn’t quite correct.
+Any Quick Launch that the user has permission to view will be listed by this endpoint.
```
Comment (ash3rz) [inline api/endpoints/quick-launches.md:36]: Should we update the Swagger docs to reflect this instead?

**Review by ash3rz — APPROVED** [review]
Wording seemed clear to me, but did have a minor question about the different wording here vs swagger. :shipit:

File: api/endpoints/quick-launches.md (hunk 1 above)
Comment (psarando) [reply api/endpoints/quick-launches.md:36]: We probably should.

I assume that's the intent of this endpoint now, but we should probably confirm with @johnworth or @slr71 if this endpoint is supposed to work this way from now on.

Comment (slr71) [reply api/endpoints/quick-launches.md:36]: As far as I know, the endpoint is supposed to continue working the way it currently does. I suspect that the Swagger docs were written before we decided to make Quick Launches sharable.

**Review by slr71 — APPROVED** [review]
LGTM. I didn't find anything to mention other than the point that @ash3rz made.

**Comment by psarando** [comment]
Thanks for the reviews! 👍 

We can update these docs and the endpoint swagger docs at the same time, but in the meantime I'll go ahead and merge this now.

**Pull request merged**